homeassistant/components/dyson/air_quality.py

```diff
@@ -1,11 +1,12 @@
 """Support for Dyson Pure Cool Air Quality Sensors."""
 import logging
 
+from libpurecool.dyson_pure_cool import DysonPureCool
```

Comment:
Please move this back to within functions again. Sorry. We've found a a case where it gives problems to import requirements at the top of the module. We're working on a fix. But let's wait for that until we move imports out.

Reply:
Done. Np I would like to refactor the code for the older models when this is merged anyway, I can move them then.